### DISCUSSION TITLE
Retrieving data frame back from openbb.stocks.options.info

### DISCUSSION THREAD

**Discussion by ori-maci** [OpenBB-finance/OpenBB#4196] [Q&A]
Hello., I was hoping someone could lead me in the right direction. I am looking to use the OpenBB sdk to retrieve back the Implied Volatility that I see in the terminal.

<img width="398" alt="image" src="https://user-images.githubusercontent.com/10456511/217968997-56f794f2-2d6a-40a5-ba13-d709c499db0d.png">

Currently, the [API](https://docs.openbb.co/sdk/reference/stocks/options/info) for the SDK does not allow me to do this but I can see the Implied Volatility in the terminal. Is there a way to retrieve this value back from the SDK? 

Here is the API usage: `options_df_data = openbb.stocks.options.info(symbol='AXP')`

**Comment by jmaslek** [comment]
Whoops :(

Great catch here.  We are mapping the wrong function to that endpoint.  Ill fix on dev now then it'll be updated in the next release!

**Reply by ori-maci** [reply]
Hello James 👋🏼 

Awesome, I really appreciate this! I will keep an eye out for the next release

**Reply by ori-maci** [reply]
Hello again @jmaslek is this change in the next PR Release? https://github.com/OpenBB-finance/OpenBBTerminal/pull/4237/files

I can't seem to find or see the change :)

**Reply by jmaslek** [reply]
Yes!  I am putting everything together now

**Reply by ori-maci** [reply]
I can confirm this now works. Thank you @jmaslek !
